Docs/source/highlights.rst

```diff
@@ -65,9 +65,12 @@ Scientific works in particle and beam modeling.
 Microelectronics
 ****************
 
-.. note::
+`ARTEMIS (Adaptive mesh Refinement Time-domain ElectrodynaMIcs Solver) <https://ccse.lbl.gov/Research/Microelectronics/>`__ is based on WarpX and couples the Maxwell's equations implementation in WarpX with classical equations that describe quantum material behavior (such as, LLG equation for micromagnetics and London equation for superconducting materials) for quantifying the performance of `next-generation microelectronics <https://www.lbl.gov/research/microelectronics-and-beyond/>`__.
 
-  Link ARTEMIS publications.
+#. Sawant SS, Yao Z, Jambunathan R, Nonaka A.
+   **Characterization of Transmission Lines in Microelectronic Circuits Using the ARTEMIS Solver**.
+   IEEE Journal on Multiscale and Multiphysics Computational Techniques, vol. 8, pp. 31-39, 2023.
+   `DOI:10.1109/JMMCT.2022.3228281 <https://doi.org/10.1109/JMMCT.2022.3228281>`__
 
 
```

Comment:
Kumar, P, Nonaka, A, Jambunathan, R, Pahwa, G and Salahuddin, S, Yao, Z. 
**FerroX: A GPU-accelerated, 3D Phase-Field Simulation Framework for Modeling Ferroelectric Devices**. 
arXiv preprint, 2023. 
`arXiv:2210.15668 <https://doi.org/10.48550/arXiv.2210.15668>`__

Yao Z, Jambunathan R, Zeng Y, Nonaka A. 
**A Massively Parallel Time-Domain Coupled Electrodynamics–Micromagnetics Solver**. 
The International Journal of High Performance Computing Applications. 2022;36(2):167-181. `DOI:10.1177/10943420211057906 <https://doi.org/10.1177/10943420211057906>`__